### ISSUE TITLE
Ranked votes are not appearing

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoIssues#8648]
I submitted a ranked vote ranking two people. 
![image](https://user-images.githubusercontent.com/3536496/42977483-55f43a0c-8b7c-11e8-9ec2-23e52983dca4.png)

It returns it properly from the 'Votes' command:
![image](https://user-images.githubusercontent.com/3536496/42977289-2757662a-8b7b-11e8-939b-538d019e5427.png)

However the rank is not displayed in the 'All Votes' window:
![image](https://user-images.githubusercontent.com/3536496/42977307-3fef6af2-8b7b-11e8-9d08-c2c876b4b653.png)
You can see my #2 ranking of '123' is not displayed.

**Comment by johnkslg** [comment]
This requires a fix I comitted to Master btw

**Comment by johnkslg** [comment]
"There are no  problems in JS. The results of the election are sent to the server like this:  {"Voter": "TestUser","RankedVotes": ["TestUser","TestUser1"]}
Then as we try to get the answers of the election we receive this answer: {"Voter": "string","RankedVotes": ["TestUser"]}
It means something is wrong in the server. I have tried with test server if the answer is received correct, the results of the election will be displayed correctly.
Also I have worked on the green tasks and have pushed the completed tasks to Feature/AC-1-UIRedo branch."
